pkgloader-recommended.sty

```diff
@@ -172,3 +172,20 @@
 %    \end{macrocode}
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  Rules from tex.stackexchange answer
+%  <https://tex.stackexchange.com/a/64457/9075>:
+%
+%    \begin{macrocode}
+\Load {mmap} before  {fontenc}
+                 if loaded
+                 because {mmap otherwise complains}
```

Comment:
Why does it complain? Let's get to the actual reason for this loading order. This could be as general as 'it is a dependency' or even 'the author recommends it'. But 'it otherwise complains' is a bit vague.

Reply:
The output of mmap/cmap is as follows:

> Package cmap Warning: fontenc already loaded - some fonts may be unprocessed.

Minimum example:

    \documentclass{article}
    \usepackage[T1]{fontenc}
    \usepackage{mmap}
    \begin{document}
    Test
    \end{document}

The documentation of CMAP (http://texdoc.net/pkg/cmap) states:

>   added warnings if cmap is loaded after fontenc or babel

Do you find time to dive into cmap's source code and try to understand its inner workings etc? If not, can we just remove the "because" statement?

Reply:
How about `because {otherwise some fonts may be unprocessed}`, as the warning tells us? Actually, let's make it `because {otherwise some fonts may be unprocessed (cmap warning)}`.